client/my-sites/promote-post-i2/components/tsp-banner/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import ExternalLink from 'calypso/components/external-link';
```

Comment:
This fails to build after merging to `trunk` because `components/external-link` was removed by @tyxla in #99594. Will be easy to fix.